CHANGELOG.md

```diff
@@ -1,3 +1,139 @@
+## 0.16.0rc1 / 2015-09-XX
+
+BREAKING CHANGES:
+
+* Release tarballs now contain the built binaries in a nested directory.
+* The `hash_mod` relabeling action now uses MD5 hashes instead of FNV hashes to
+  achieve a better distribution.
+* The DNS-SD meta label `__meta_dns_srv_name` was renamed to `__meta_dns_name`
+  to reflect support for other record types than `SRV`.
+* The default full refresh interval for the file-based service discovery has been
+  increased from 30 seconds to 5 minutes.
+* In relabeling, parts of a source label that weren't matched by
+  the specified regular expression are no longer included in the replacement output.
+* Queries no longer interpolate between two data points. Instead, the resulting
+  value will always be the latest value before the evaluation query timestamp.
+* Any regular expressions (in `label_replace()` or supplied via the
+  configuration) are now anchored to match full strings instead of substrings.
+* Global labels are not appended upon storing time series anymore. Instead,
+  they are only appended when communicating with external systems
+  (Alertmanager, remote storages, federation).
+* The names and units of metrics related to remote storage sample appends have
+  been changed.
+* The experimental support for writing to InfluxDB has been updated to work
+  with InfluxDB 0.9.x. 0.8.x versions of InfluxDB are not supported anymore.
+
+All changes:
+
+* [CHANGE] Vendoring is now done via govendor instead of godep.
```

Comment:
Perhaps mention that we require Go 1.5.x now and the GO15VENDOREXPERIMENT=1 env variable.